CMakeLists.txt

```diff
@@ -968,7 +968,14 @@ add_subdirectory(contrib)
 add_subdirectory(src)
 
 if(BUILD_TESTS)
-  add_subdirectory(tests)
+  if(CMAKE_BUILD_TYPE STREQUAL "fuzz")
+    if (UNIX)
+      set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} -pthread")
```

Comment:
That doesn't seem right. cmake already tests for threading in ${CMAKE_THREAD_LIBS_INIT}. Maybe this doesn't like against that. Everything should I think, in our case.

Reply:
Agreed, I thought this was strange as well, but without this change the build fails. With the change it succeeds.